arcgis-ios-sdk-samples/Maps/Manage operational layers/MMLLayersViewController.swift

```diff
@@ -15,150 +15,149 @@
 import UIKit
 import ArcGIS
 
-protocol MMLLayersViewControllerDelegate: AnyObject {
-    func layersViewControllerWantsToClose(_ layersViewController: MMLLayersViewController, withDeletedLayers layers: [AGSLayer])
-}
-
 class MMLLayersViewController: UITableViewController {
-    var layers: NSMutableArray!
-    var deletedLayers: [AGSLayer]!
     
-    weak var delegate: MMLLayersViewControllerDelegate?
```

Comment:
I can understand (a little bit) not wanting to take the time to implement a delegate. What I don't understand is removing a delegate that already exists and replacing it with a closure property.

Reply:
It's never been about not wanting to take the time. I've told you before that I don't like delegates when there are only one or two callbacks. I changed it to a closure because I think it is the better solution.

Reply:
It seems like we've been repeating discussions over the same coding differences. I think a lot of our disagreements could be avoided if Esri (or even just Runtime) had comprehensive Swift guidelines. Both of our methods work, so lacking a standard I don't feel compelled to code a particular way.

Reply:
In the case of this sample, I pushed new changes that removes the need for a callback at all.